pci_info.py

```diff
@@ -109,4 +110,5 @@ def create_config(pci_list):
     if args.show_info:
         pprint(pci_details)
     if args.create_cfg:
-        create_config(pci_details)
+        cmd = create_config(pci_details)
+        print cmd
```

Comment:
are we really need to print cmd ?

Reply:
Yes. In case the user wants to run tests on his own, this needs to be printed.

Reply:
ah ok